src/test/java/org/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -13,16 +14,20 @@
 import org.jabref.model.entry.types.StandardEntryType;
 import org.jabref.testutils.category.FetcherTest;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 @FetcherTest
-class ArXivTest {
+class ArXivTest implements SearchBasedFetcherCapabilityTest {
     private ArXiv finder;
```

Comment:
Isn't `ArXiv` a `Fetcher`? - I think, this refactoring can be done here, too.

```suggestion
    private ArXiv fetcher;
```